virtme_ng/version.py

```diff
@@ -37,9 +48,9 @@ def get_version_string():
 
         return version_pep440
     except CalledProcessError:
-        # Default version if git describe fails (e.g., when building virtme-ng
-        # from a source package.
-        return PKG_VERSION
+        # If git describe fails to determine a version, try to use the version
+        # from pip, or ultimately simply return the hard-coded package version.
+        return get_package_version()
```

Comment:
Is it safe to fallback to the version from pip here if you have a "local" version? I mean: if we are here, it means we are not using a version from pip from what I understood, then we might be looking at a wrong (old) version, no?

Reply:
Indeed! In this case we definitely want to use `PKG_VERSION`, good catch (feel free to send a PR if you want, otherwise I'll fix it later)

Reply:
> feel free to send a PR if you want

done! #139